.github/workflows/covector.yml

```diff
@@ -0,0 +1,150 @@
+name: Covector
+on:
+  push:
+    branches:
+      - production
+    paths-ignore:
+      - 'specs/**'
+      - 'documentation/**'
+
+jobs:
+  status:
+    name: Status
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the Source Code
+        uses: actions/checkout@v2
+
+      - name: Run Covector Status
+        uses: jbolda/covector/packages/action@covector-v0
+        id: covector
+        with:
+          command: status
+
+  version-or-publish:
+    name: Version-or-publish
+    runs-on: ubuntu-latest
+    outputs:
+      change: ${{ steps.covector.outputs.change }}
+      commandRan: ${{ steps.covector.outputs.commandRan }}
+      successfulPublish: ${{ steps.covector.outputs.successfulPublish }}
+
+    steps:
+      - name: Checkout the Source Code
+        uses: actions/checkout@v2
+
+      - name: Set Up Node.js 14
+        uses: actions/setup-node@v2
```

Comment:
Btw there is now a v3 of this action: https://github.com/actions/setup-node/releases/tag/v3.0.0